docs/io/fits/usage/headers.rst

```diff
@@ -102,8 +102,8 @@ You can also add a new value *and* comment by assigning them as a tuple::
         >>> del hdr['DARKCORR']  # Delete previous insertion for doctest
         >>> hdr.insert(20, ('DARKCORR', 'OMIT', 'Dark Image Subtraction'))
 
-      This inserts the DARKCORR keyword before the 20th keyword in the header
-      no matter what it is.
+      This inserts the DARKCORR keyword before the twentieth keyword in the
```

Comment:
I would leave this as the number because it's then easier to see the correspondence between the descriptive text and the code above
```suggestion
      This inserts the DARKCORR keyword before the 20th keyword in the
```